pages/docs/configuration/configuration-communication.md

```diff
@@ -44,11 +44,11 @@ The alternative to TCP/IP sockets is MPI ports (an MPI 2.0 feature):
 <m2n:mpi .../>
 ```
 
-As the ports functionality is not a highly used feature of MPI, it has robustness issues for several MPI implementations ([for OpenMPI, for example](https://github.com/precice/precice/issues/746)). In principle, MPI gives you faster communication roughly by a factor of 10, but, for most applications, you will not feel any difference as both are very fast. We recommend using `sockets`.
+As the ports functionality is not a highly used feature of MPI, it has robustness issues for several MPI implementations ([for OpenMPI, for example](https://github.com/precice/precice/issues/746)). In principle, MPI gives you faster communication roughly by a factor of 10 (see [Benjamin Uekermann's dissertation](https://mediatum.ub.tum.de/doc/1320661/document.pdf), section 4.2.3), but, for most applications, you will not feel any difference as both are very fast. We recommend using `sockets` by default, unless you are performing a large performance-critical simulation.
```

Comment:
```suggestion
As the ports functionality is not a highly used feature of MPI (if used with separated `MPI_COMM_WORLD` communicators), it has robustness issues for several MPI implementations ([for OpenMPI, for example](https://github.com/precice/precice/issues/746)). In principle, MPI gives you faster communication roughly by a factor of 10 (see [Benjamin Uekermann's dissertation](https://mediatum.ub.tum.de/doc/1320661/document.pdf), section 4.2.3), but, for most applications, you will not feel any difference as both are very fast. We recommend using `sockets` by default, unless you are performing large performance-critical simulations with very large coupling meshes.
```

Reply:
I think the "separated `MPI_COMM_WORLD` communicators" will still not be clear for many users. Can we maybe give an example or comment how probable it is that it affects a user?

Reply:
preCICE participants are typically always started with separated `MPI_COMM_WORLDS`. This first sentence here is a bit to blame MPI.
Maybe linking to https://precice.org/running-slurm.html helps?

Our main message should however always be: "use sockets, it's not really slower, but much more robust".